src/core/plugins/spec/actions.js

```diff
@@ -230,7 +232,7 @@ const debResolveSubtrees = debounce(async () => {
           }))
       }
       set(resultMap, path, spec)
-      set(specWithCurrentSubtrees, path, spec)
+      specWithCurrentSubtrees = assocPath(path, spec, specWithCurrentSubtrees)
```

Comment:
We don't want to change the `specWithCurrentSubtrees` object reference unless it's absolutely necessary.